src/hooks/useANJBalanceToUsd.js

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from 'react'
+import { getMarketDetails, getTokenReserves } from '@uniswap/sdk'
+import { useCourtConfig } from '../providers/CourtConfig'
+
+import env from '../environment'
+import { getDaiAddress } from '../utils/known-tokens'
+import { formatUnits, bigNum } from '../lib/math-utils'
+import { addressesEqual, ETH_FAKE_ADDRESS } from '../lib/web3-utils'
+
+const UNISWAP_PRECISION = 18
+const UNISWAP_MARKET_RETRY_EVERY = 1000
+
+async function getANJMarketDetails(tokenAddress, anjTokenAddress) {
+  const [tokenData, anjData] = await Promise.all(
+    [tokenAddress, anjTokenAddress].map(address => {
+      // In the case of ETH, undefined should be passed as the reserves data.
+      if (addressesEqual(address, ETH_FAKE_ADDRESS)) {
+        return undefined
+      }
+
+      const tokenReserves = getTokenReserves(address, env('CHAIN_ID'))
+
+      if (!tokenReserves) {
+        throw new Error('Could not fetch reserves')
```

Comment:
Not sure if here should throw an error that will be catched by the `GlobalError` component, i think that we should not  show a global error if we just couldn't fetch from uniswap

Reply:
In this case it would be catched by the `try {} catch() {}` in `useANJBalanceToUsd `